test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -169,7 +181,9 @@ public static String changeOrgAndTag(String image) {
         String dockerOrg = System.getenv().getOrDefault("DOCKER_ORG", strimziOrg);
         String dockerTag = System.getenv().getOrDefault("DOCKER_TAG", strimziTag);
         kafkaVersion = System.getenv().getOrDefault("KAFKA_VERSION", kafkaVersion);
-        return changeOrgAndTag(image, dockerOrg, dockerTag, kafkaVersion);
+        String imagePrefix = System.getenv().getOrDefault("DOCKER_IMAGE_PREFIX", null);
```

Comment:
why are you using `null` instead of empty string there? In case of empty string you can skip some checks in `changeOrgAndTag` method